UsersGitHub/UsersGitHub/Services/UserService.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Refit;
+using UsersGitHub.Interfaces;
+using UsersGitHub.Model;
+using UsersGitHub.Model.DtoModel;
+using Xamarin.Forms.Internals;
+
+namespace UsersGitHub.Services
+{
+    public class UserService
+    {
+        public async Task<string> GetUserInfo(string userName)
+        {
+            UserDto user;
+            var httpClient = new HttpClient
```

Comment:
you have a code duplication here when creating gitHubApi. it's better to implement a getter and initialize it in constructor